src/app.js

```diff
@@ -27,11 +25,8 @@ app.use(express.urlencoded({ extended: false }));
 
 // Setup Swagger.
 // Don't use `/` for swagger, it will catch everything.
-const spec = parseComments({
-  definition: baseDefinition,
-  paths: [path.join(__dirname, '**/*.?(js|yaml|yml)')],
-});
-app.use("/api-docs", swaggerUi.serve, swaggerUi.setup(spec));
+const specs = openapi();
```

Comment:
I think we should use this to be on the safe side:
```js
const spec = openapi({ cwd: __dirname });
```

Reply:
Updated